web/impact/impact/v1/helpers/organization_helper.py

```diff
@@ -0,0 +1,67 @@
+from impact.models import Organization
+from impact.v1.helpers.model_helper import ModelHelper
+
+
+class OrganizationHelper(ModelHelper):
+    MODEL = Organization
+
+    REQUIRED_KEYS = [
+        "name",
+        "url_slug",
+        ]
+    OPTIONAL_KEYS = [
+        "additional_industry_ids",
+        "date_founded",
+        "facebook_url",
+        "full_elevator_pitch",
+        "primary_industry_id",
+        "public_inquiry_email",
```

Comment:
why are these keys not derived from the "required=true" attribute on the field itself? seems like we'd have to maintain this in 2 or 3 places now (impact->django-accelerator->accelerate and here)

Reply:
See comment at the end.